spec.bs

```diff
@@ -1060,27 +1060,40 @@ To <dfn>obtain the plaintext payload</dfn> given an [=aggregatable report=]
 <div algorithm>
 To <dfn>encrypt the payload</dfn> given a [=byte sequence=] |plaintextPayload|,
 public key |pkR| and a [=string=] |sharedInfo|, perform the following steps.
-They either return a [=byte sequence=] or throw an error.
+They return a [=byte sequence=] or an error.
 
 1. Let |info| be the result of [=UTF-8 encoding=] the [=string/concatenation=]
     of « "`aggregation_service`", |sharedInfo| ».
-1. Let |aad| be \`\` (an empty [=byte sequence=]).
-1. Let |ciphersuite| be (0x0020, 0x0001, 0x0003).
+1. Let (|kem_id|, |kdf_id|, |aead_id|) be (0x0020, 0x0001, 0x0003).
 
-    Note: The |ciphersuite| triple is composed of HPKE
+    Note: The ciphersuite triple above is composed of [[RFC9180|HPKE]]
         [[RFC9180#name-algorithm-identifiers|algorithm identifiers]], specifying
         the [[RFC9180#name-key-encapsulation-mechanism|KEM]] as DHKEM(X25519,
         HKDF-SHA256), the [[RFC9180#name-key-derivation-functions-kd|KDF]]
         function as HKDF-SHA256 and the
         [[RFC9180#name-authenticated-encryption-wi|AEAD]] function as
         ChaCha20Poly1305.
-1. Let |encryptedPayload| be the result of calling HPKE's
-    [[RFC9180#single-shot-apis|single-shot encryption function]]: <var
-    ignore>Seal&lt;Base&gt;</var>(|pkR|, |info|, |aad|, |plaintextPayload|).
-    Although RFC9180 omits the parameter from its pseudocode, |ciphersuite| is
-    passed implicitly.  All of RFC9180's high-level HPKE operations
-    [[RFC9180#name-errors|are fallible]]; if encryption fails, throw
-    [=exception/throw=] a {{TypeError}}.
+1. Let (|enc|, |hpkeContext|) be the result of setting up an [[RFC9180|HPKE]]
+    [[RFC9180#name-encryption-to-a-public-key|sender's context]] by calling <var
+    ignore>SetupBaseS()</var> with a public key |pkR|, application-supplied
```

Comment:
nit: Should SetupBaseS() be in backticks (i.e. monospace font) instead of treating as a variable (italics)?

ditto for Seal() below (and the other SetupBaseS())

Reply:
Sure, that works. I have no idea what the convention is for referencing familiar categories of objects that are defined outside of the Bikeshed bubble.

The mixing and matching is still a little weird, but I think it's unavoidable. Rather than saying "call *hpkeContext.Seal()* with ...", now I'll say "call `ContextS.Seal()` on the *hpkeContext* object with ...."